biocrnpyler/propensities.py

```diff
@@ -167,47 +167,122 @@ def from_dict(cls, propensity_dict):
         merged.update(propensity_dict['species'])
         return cls(**merged)
 
-    def _create_bioscrape_annotation(self, model, propensity_dict_in_sbml):
+    def _create_annotation(self, model, propensity_dict_in_sbml, **kwargs):
+        '''
```

Comment:
Usually, triple double-quote is used for docstring: """, but you can leave it like this if you want.

Reply:
Fixed.